.github/workflows/canary.yml

```diff
@@ -2,7 +2,7 @@ name: "Canary Build"
 
 on:
   schedule:
-    - cron: '0 0 * * 0'  # Run weekly on Sunday at midnight
+    - cron: "0 0 * * 0" # Run weekly on Sunday at midnight
```

Comment:
Is this still needed?  With the latest setup we had these canary builds run with every PR, not a schedule.

Reply:
The current behavior is doing it on both. We run it every Sunday night AND on every PR. If we are happy just doing it on PRs we can drop the `schedule` block.